tests/expectations/metrics/test_core.py

```diff
@@ -1991,9 +1991,7 @@ def test_map_column_values_increasing_pd():
     )
     metrics.update(results)
 
-    assert metrics[unexpected_rows_metric.id]["a"].index == pd.Int64Index(
-        [4], dtype="int64"
-    )
+    assert metrics[unexpected_rows_metric.id]["a"].index == pd.Index([4], dtype="int64")
```

Comment:
Will these fail on our earliest supported pandas versions? If so we may need to bifurcate based on installed pandas version, if not then 👍

Reply:
Yes this will work with earlier versions 👍🏼